events/2021-virtual-beer-v0.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Virtual Beer v0
+---
+
+## È tempo della prima Virtual Beer!
+
+Stavamo organizzando il nostro **8° apericoder** quando è scoppiata la pandemia
+e le attività del nostro gruppo hanno ricevuto un forzato stop. L'idea iniziale
+era che un gruppo nato per far incontrare professionisti del software vicini fra
+loro avrebbe giovato poco spostando le attività esclusivamente online.
+
+Nonostante tutto, il nostro **Slack** e il gruppo **Telegram** sono rimasti
+timidamente attivi ed è nata nell'ultima settimana anche un interesse ad
+organizzare un incontro virtuale in modalità disinvolta, con argomento iniziale
+su **GIT e branching model agile**, che potrà evolversi o cambiare completamente
+a seconda di come la chiacchierata si evolverà.
```

Comment:
evolversi, evolverà: non mi piace questa ripetizione di "evolvere", ma non mi viene in mente di meglio

Reply:
forse si potrebbe scrivere: "su cui potremo dilungarci o cambiare completamente"?